tests/Unit/DataStructures/VectorImplTestHelper.hpp

```diff
@@ -226,12 +147,12 @@ void vector_test_construct_and_assign(
   move_assignment_initialized = std::move(initializer_list_constructed_copy);
   CHECK(move_assignment_initialized.is_owning());
 
-  const VectorType move_constructed{std::move(move_assignment_initialized)};
```

Comment:
You're removing a lot of `const`. Is this some relic of the git history or something intentional?

Reply:
Yeah, I think this a git trouble. More reason to split off at this point and have a cleaner set of commits.